tests/unit/challenges/test_views.py

```diff
@@ -185,6 +185,29 @@ def test_particular_challenge_host_team_for_challenge_does_not_exist(self):
         self.assertEqual(response.status_code, status.HTTP_406_NOT_ACCEPTABLE)
 
 
+class GetParticipantTeamNameTest(BaseAPITestClass):
+
+    def setUp(self):
+        super(GetParticipantTeamNameTest, self).setUp()
+
+        self.participant = Participant.objects.create(
+            user=self.user,
+            status=Participant.ACCEPTED,
+            team=self.participant_team
+        )
+
+        self.challenge.participant_teams.add(self.participant_team)
+
+    def test_team_name_for_challenge(self):
```

Comment:
@Kajol-Kumari  can you add a test for the case when request is not from a participant team